### PR TITLE
Adding models from schema

### PR DESCRIPTION
**Pull request by ivanboika**
I've added all models from schema without extra changes. I remember that @132dannys task may include changes at schema as optimization process
Also, I'm not quite sure that I've stored models properly

### CODE REVIEW

File: api/anri/apps/products/models/order.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 from django.db import models
 
 from anri.apps.common.models import CoreModel
-from anri.apps.products.models.user import User
 
 
 class Order(CoreModel):
-    user_id = models.OneToOneField(User.uuid)
+    user_id = models.OneToOneField("User")
```
Comment (132dannys) [inline api/anri/apps/products/models/order.py:7]: user_id - >user

**Review by Orange-hanter — CHANGES_REQUESTED** [review]

File: api/anri/apps/users/models/contragent.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from anri.apps.common.models import CoreModel
+
+
+class Contragent(CoreModel):
+    company_name = models.CharField(max_length=127)
+    state_cadastr_address = models.CharField(max_length=127)
+    legal_address = models.CharField(max_length=127)
+    company_rating = models.DecimalField(max_digits=5, decimal_places=2)
+    contact_name = models.CharField(max_length=127)
+    contact_email = models.CharField(max_length=127)
```
[the remark this answers is not in the document]
Comment (ivanboika) [reply api/anri/apps/users/models/contragent.py]: Done

File: api/anri/apps/orders/models/order_item.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from anri.apps.common.models import CoreModel
+
+
+class OrderItem(CoreModel):
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    # product = models.ForeignKey("products.Product", on_delete=models.CASCADE)
+    quantity = models.IntegerField()
```
[the remark this answers is not in the document]
Comment (ivanboika) [reply api/anri/apps/orders/models/order_item.py]: Comment fixed, where is typo I don't see

File: api/anri/apps/carts/models/cart_item.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.conf import settings
+
+from anri.apps.common.models import CoreModel
+
+
+class CartItem(CoreModel):
+    cart = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    # product = models.ForeignKey("products.Product", on_delete=models.CASCADE)
+    quantity = models.IntegerField()
```
[the remark this answers is not in the document]
Comment (ivanboika) [reply api/anri/apps/carts/models/cart_item.py]: Done

File: api/anri/apps/carts/models/cart_item.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.conf import settings
+
+from anri.apps.common.models import CoreModel
+
+
+class CartItem(CoreModel):
+    cart = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    # product = models.ForeignKey("products.Product", on_delete=models.CASCADE)
```
[the remark this answers is not in the document]
Comment (ivanboika) [reply api/anri/apps/carts/models/cart_item.py]: Fixed

**Review by Orange-hanter — APPROVED** [review]

**Comment by Orange-hanter** [comment]
@132dannys approve please

**Review by 132dannys — APPROVED** [review]

**Pull request merged**